File: AngularMedicalGroup/src/app/models/provider.model.ts
export class Provider{
    public basicinformation:BasicInformation;
    public providerInformation:ProviderInformation;
    public clia:Clia;
    public authConfiguration:AuthConfiguration;
    public letterDispatchMode:LetterDispatchMode;
    public primaryLocation:PrimaryLocation;
    public appointmentStatus:AppointmentStatus;
    public contactStatus:ContactStatus;
    public organizationInformation:OrganizationInformation[];
}
export class BasicInformation{
    public category:string;
    public npi:string;
    public providertype:string;
    public providerid:string;
    public primaryspeciality:string;
    public preferedspeciality:boolean;
    public referanymember:boolean;
    public nameTitle:string;
    public nameLast:string;
    public nameFirst:string;
    public nameMiddle:string;
    public nameSuffix:string;
    public nameProfSuffix:string;
    public nameMiddleName:string;
    public personalGender:string;
    public personalDateOfBirth:string;
    public personalSsn:string;
    public personalCitezenship:string;
    public personalBirthCity:string;
    public personalBirthState:string;
    public personalBirthCountry:string;
}
export class ProviderInformation{
    public contractType:string;
    public fromDate:string;
    public toDate:string;
    public paymentType:string;
    public account:string;
    public carrierlocality:string;
}
export class Clia{
    public certified:boolean;
    public waivable1:boolean;
    public waivable2:boolean;
}
export class AuthConfiguration{
    public validfrom:string;
    public allowedVisits:string;
    public acceptPatient:boolean;
    public allowasReferingProvider:boolean;
}
export class LetterDispatchMode{
    public authorization:string;
    public caseManagement:string;
    public claims:string;
    public clinicalAlerts:string;
    public credentialing:string;
    public hedis:string;
    public pdr:string;
    public referalTracking:string;
}
export class PrimaryLocation{
    public practiceType:string;
    public name:string;
    public address1:string;
    public address2:string;
    public aptno:string;
    public city:string;
    public zip:string;
    public state:string;
    public countryid:string;
    public country:string;
    public phone:string;
    public fax:string;
}
export class ContactStatus{
    public contractTemplate:string;
    public fromDate:string;
    public toDate:string;
}
export class OrganizationInformation{
    public organizationId:string;
    public organizationName:string;
    public organizationType:string;
    public organizationEffectiveFrom:string;
    public organizationEffectiveTo:string;
}
export class AppointmentStatus{
    public parentPayorFacility1:string;
    public parentPayorFacility2:string;
    public appType:string;
}

File: DataBase/Models/ProviderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DataBase.Models
{
    public class ProviderModel
    {
        public BasicInformation basicinformation;
        public ProviderInformation providerInformation;
        public Clia clia;
        public AuthConfiguration authConfiguration;
        public LetterDispatchMode letterDispatchMode;
        public PrimaryLocation primaryLocation;
        public AppointmentStatus appointmentStatus;
        public ContactStatus contactStatus;
        public OrganizationInformation[] organizationInformation;
    }
    public class BasicInformation
    {
        public string category;
        public string npi;
        public string providertype;
        public string providerid;
        public string primaryspeciality;
        public bool preferedspeciality;
        public bool referanymember;
        public string nameTitle;
        public string nameLast;
        public string nameFirst;
        public string nameMiddle;
        public string nameSuffix;
        public string nameProfSuffix;
        public string nameMiddleName;
        public string personalGender;
        public string personalDateOfBirth;
        public string personalSsn;
        public string personalCitezenship;
        public string personalBirthCity;
        public string personalBirthState;
        public string personalBirthCountry;
    }
    public class ProviderInformation
    {
        public string contractType;
        public string fromDate;
        public string toDate;
        public string paymentType;
        public string account;
        public string carrierlocality;
    }
    public class Clia
    {
        public bool certified;
        public bool waivable1;
        public bool waivable2;
    }

    public class AuthConfiguration
    {
        public string validfrom;
        public string allowedVisits;
        public bool acceptPatient;
        public bool allowasReferingProvider;
    }
    public class LetterDispatchMode
    {
        public string authorization;
        public string caseManagement;
        public string claims;
        public string clinicalAlerts;
        public string credentialing;
        public string hedis;
        public string pdr;
        public string referalTracking;
    }
    public class PrimaryLocation
    {
        public string practiceType;
        public string name;
        public string address1;
        public string address2;
        public string aptno;
        public string city;
        public string state;
        public string zip;
        public string countryid;
        public string country;
        public string phone;
        public string fax;
    }
    public class ContactStatus
    {
        public string contractTemplate;
        public string fromDate;
        public string toDate;
    }
    public class OrganizationInformation
    {
        public string organizationId;
        public string organizationType;
        public string organizationEffectiveFrom;
        public string organizationEffectiveTo;
        public string organizationName;
    }
    public class AppointmentStatus
    {
        public string parentPayorFacility1;
        public string parentPayorFacility2;
        public string appType;
    }
}